python/lsst/cp/pipe/cpLinearitySolve.py

```diff
@@ -152,6 +152,13 @@ class LinearitySolveConfig(pipeBase.PipelineTaskConfig,
         doc="Number of spline knots to use in fit.",
         default=10,
     )
+
+    trimmedState = pexConfig.Field(
```

Comment:
I don't think that this is quite the right name.  But then again the docs are clear, and I don't have a better suggestion off the top of my head.